src/devices/gledopto.ts

```diff
@@ -1034,4 +1035,237 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Zigbee relay switch",
         extend: [m.onOff()],
     },
+const SCENE_DATA = {
+    'Ice land blue': [0x01, 0x15, 0x0a, 0x52, 0x52, 0xe0, 0x00, 0x00, 0x64, 0x00, 0xc1, 0x61, 0x00, 0xb4, 0x30, 0x00, 0xb5, 0x52, 0x00, 0xc4, 0x63],
+    'Glacier express': [0x01, 0x16, 0x0a, 0x64, 0x64, 0x60, 0x00, 0x00, 0x64, 0x00, 0x92, 0x5f, 0x00, 0xc6, 0x60],
+    'Sea of clouds': [0x01, 0x17, 0x03, 0x5e, 0x5e, 0x60, 0x00, 0x00, 0x64, 0x00, 0x38, 0x2f, 0x00, 0x1e, 0x5c, 0x00, 0xd5, 0x45, 0x01, 0x1a, 0x64],
```

Comment:
```suggestion
    'sea_of_clouds': [0x01, 0x17, 0x03, 0x5e, 0x5e, 0x60, 0x00, 0x00, 0x64, 0x00, 0x38, 0x2f, 0x00, 0x1e, 0x5c, 0x00, 0xd5, 0x45, 0x01, 0x1a, 0x64],
```

Make all values snake case (same for `MUSIC_DATA`)